src/monsters.cpp

```diff
@@ -765,6 +753,23 @@ bool Monsters::deserializeSpell(MonsterSpell* spell, spellBlock_t& sb, const std
 			          << " - Unknown spell name: " << spell->name << std::endl;
 		}
 
+		if (!conditionDeclared && spell->conditionType != CONDITION_NONE) {
```

Comment:
Isnt it enough to move this as last else if?
Then just remove that condition declared